src/components/InputField.tsx

```diff
@@ -63,17 +76,32 @@ class InputField extends React.Component<InputFiledProps, InputFieldState> {
         )
     }
 
-    // Make better names
-    onChange = (rate?: number) => {
+    changeHourlyRateState = (rate?: number) => {
         // this.setState({ [e.target.name]: e.target.value });
         this.setState({ hourlyRate: rate });
```

Comment:
a tip to shorten these helper functions for updating state is to use es6 object construction syntax, where a property can be assigned directly if it has the same name as a variable:
`changeHourlyRateState = (hourlyRate?: number) => this.setState({hourlyRate})`